lib/progress.coffee

```diff
@@ -71,7 +70,7 @@ getProgressStream = (total, onState = noop) ->
 #		stream.on 'progress', (state) ->
 #			console.log(state)
 ###
-exports.estimate = (requestAsync) -> (options) ->
+exports.estimate = (requestAsync, isBrowser) -> (options) ->
```

Comment:
This is interesting, because this is a curried function, with two separate set of params. I've changed the format, and right now the `@param` references the latter, so I think this is ok as is? It's pretty debatable what `@param` should be used here though.

Reply:
Oh, I didn't realize that this was curried...
SGTM
PS: I'm not experienced on jsdoc for curried fns but since this isn't exposed I guess there isn't a need to overanalyze this.